identity-core/Cargo.toml

```diff
@@ -34,15 +33,15 @@ version = "0.7"
 default-features = false
 features = ["blake2b", "ed25519", "random", "sha"]
 
+[target.'cfg(target_arch = "wasm32")'.dependencies]
```

Comment:
I think this also needs `not(target_os = "wasi")` ?

Reply:
Thanks for the catch, don't know why I missed this twice :man_facepalming:.